web/src/routes/results/[eid]/+page.svelte

```diff
@@ -0,0 +1,259 @@
+				  )
+				: [cur],
+		};
+	}, {} as { [key: string]: Array<ElectionOutcomeResult> });
+	$: winner = electionOutcome.results
+		.filter((e) => e.isElected)
+		.map((e) => e.name)
+		.join(", ");
+	$: date = new Date(electionOutcome.date).toUTCString();
+
+	let deleteElectionDialog: HTMLDialogElement;
+	const deleteElection = async (id: number) => {
+		$fetching = true;
+		const response = await fetch(API.ADMIN_ELECTION, {
+			method: "DELETE",
+			body: JSON.stringify({ id }),
+		});
+
+		if (response.ok) {
+			elections.set(await _getElections());
+			goto("/");
+		} else {
+			$error = new Error(await response.text());
+		}
+		$fetching = false;
+	};
+
+	const publishElectionResults = async (id: number, published: boolean) => {
+		$fetching = true;
+		const response = await fetch(API.ADMIN_ELECTION_PUBLISH, {
+			method: "POST",
+			body: JSON.stringify({ id, published }),
+		});
+
+		if (response.ok) {
+			electionOutcome.isPublished = published;
+		} else {
+			$error = new Error(await response.text());
+		}
+		$fetching = false;
+	};
+</script>
+
+<svelte:head>
+	<title>Outcome of election for {election.roleName}</title>
+</svelte:head>
+
+<Panel title={`Outcome of election for ${election.roleName}`}>
+	<p>{election.description}</p>
+</Panel>
+
+{#if $user.isAdmin && !electionOutcome.isPublished}
+	<Panel kind="emphasis">
+		<div class="unpublished-callout">
+			<p>
+				The results for <b>{election.roleName}</b> have not been published. While they remain unpublished,
+				only administrators may view the results.
+			</p>
+			<div class="button">
+				<Button
+					kind="primary"
+					text="Publish results"
+					on:click={() => publishElectionResults(election.id, true)}
+				/>
+			</div>
+		</div>
+	</Panel>
+{/if}
+
+<Panel title="Results" headerIcon="receipt_long">
+	<div class="results">
+		<p>
+			This election was held at <b>{date}</b>. There were {electionOutcome.ballots}
+			ballots cast and {electionOutcome.rounds} rounds in total.
+		</p>
+		{#each Object.entries(resultsByRound) as [round, results]}
+			<h3>Round {round}</h3>
+			<table>
+				<thead>
+					<tr>
+						<th>Candidate</th>
+						<th>Votes</th>
+						<th>Status</th>
+					</tr>
+				</thead>
+				<tbody>
+					{#each results as result}
+						<tr>
+							<td>{result.name}</td>
+							<td>{result.voteCount}</td>
+							<td>
+								{#if result.isRejected}
+									Rejected
+								{:else if result.isElected}
+									Elected
+								{:else}
+									Proceed
+								{/if}
+							</td>
+						</tr>
+					{/each}
+				</tbody>
+			</table>
+			{#if results.some((e) => e.isRejected)}
+				<p>
+					<b>
+						{results
+							.filter((e) => e.isRejected)
+							.map((e) => e.name)
+							.join(", ")}
+					</b> was eliminated
+				</p>
+			{/if}
+		{/each}
+		<p>
+			After {electionOutcome.rounds} rounds of voting, the winning candidate for the post of
+			<b>{election.roleName}</b>
+			is <b>{winner}</b>
+		</p>

```

Comment:
Seeing as this is the most important bit of info on the page, IMO it should be at the top of the panel

Reply:
Maybe, though in my mind the page is supposed to be read as a sequence of events in order (like it is being announced)

Reply:
Ok, that's fair 👍